tests/unit/modules/test_xml.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+'''
+    Tests for xml module
+'''
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+import tempfile
+
+from salt.modules import xml
+
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.unit import TestCase, skipIf
+from tests.support.mock import (
+    NO_MOCK,
+    NO_MOCK_REASON
+)
+
+XML_STRING = '''
+    <root xmlns:foo="http://www.foo.org/" xmlns:bar="http://www.bar.org">
+        <actors>
+            <actor id="1">Christian Bale</actor>
+            <actor id="2">Liam Neeson</actor>
+            <actor id="3">Michael Caine</actor>
+        </actors>
+        <foo:singers>
+            <foo:singer id="4">Tom Waits</foo:singer>
+            <foo:singer id="5">B.B. King</foo:singer>
+            <foo:singer id="6">Ray Charles</foo:singer>
+        </foo:singers>
+    </root>
+    '''
+
+
+@skipIf(NO_MOCK, NO_MOCK_REASON)
```

Comment:
Doh - one more?

(also, perhaps fire off a `grep -Inr NO_MOCK tests/` to make sure we're not missing any others)

Reply:
now no_mock clean

Reply:
[so](https://jenkinsci.saltstack.com/job/pr-lint/job/PR-54977/10/pylint/folder.592290234/) [close](https://jenkinsci.saltstack.com/job/pr-lint/job/PR-54977/10/pylint/folder.10734287/) (the skipIf imports are now unused 😬 )

We'll get there! 😂

Reply:
Things are slightly better today than they were yesterday ;)